plugins/filter_kubernetes/kube_conf.c

```diff
@@ -61,6 +62,19 @@ struct flb_kube *flb_kube_conf_create(struct flb_filter_instance *ins,
         return NULL;
     }
 
+#ifdef FLB_HAVE_KUBE_TOKEN_COMMAND
+    /* K8s Token Command */
+    cmd = flb_filter_get_property("kube_token_command", ins);
```

Comment:
this must be handled in the config map.

Reply:
see the comment on the config map, setting from the config map didn't work if the command has spaces in it. But using this option did work.

Reply:
ack